docusaurus/docs/Android/02-client/06-guides/07-chatdomain-migration.mdx

```diff
@@ -89,6 +89,11 @@ val threadState = chatClient.state.thread(messageId = "parentMessageId")
 val globalState = chatClient.globalState
 ```
 
+:::warning
+Methods mentioned above return a state object associated with the API call but don't perform the API call itself.
+Make sure to request the data using `ChatClient` object.
```

Comment:
I think we have this, but maybe a small example here would be good? E.g. show how using `channelsState and queryChannels` works together